ui/app/templates/components/oidc/provider-config-header.hbs

```diff
@@ -0,0 +1,21 @@
+<PageHeader as |p|>
```

Comment:
Goal here is to have this header component display the tabs if it _is_ a route index page:
<img width="1215" alt="Screen Shot 2022-06-21 at 11 42 18 AM" src="https://user-images.githubusercontent.com/68122737/174874511-02123702-1984-480f-bc3f-08aef5f524f8.png">
<hr>
otherwise show the title with breadcrumb nav, with a title:
<img width="934" alt="Screen Shot 2022-06-21 at 11 43 18 AM" src="https://user-images.githubusercontent.com/68122737/174874718-4794014c-506a-479e-b783-5833844db0f8.png">

 **or**